webpack/scenes/SmartProxy/__tests__/SmartProxyContentTest.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { renderWithRedux, patientlyWaitFor } from 'react-testing-lib-wrapper';
+
+import { nockInstance, assertNockRequest } from '../../../test-utils/nockWrapper';
+import api from '../../../services/api';
+import SmartProxyContentTable from '../SmartProxyContentTable';
+
+const smartProxyContentData = require('./SmartProxyContentResult.fixtures.json');
+
+const smartProxyContentPath = api.getApiUrl('/capsules/1/content/sync');
+
+const smartProxyContent = { ...smartProxyContentData };
+
+const contentTable = <SmartProxyContentTable smartProxyId={1} />;
+
+test('Can display Smart proxy content table', async (done) => {
+  const detailsScope = nockInstance
+    .get(smartProxyContentPath)
+    .query(true)
+    .reply(200, smartProxyContent);
+
+  const { getByText, queryByText } = renderWithRedux(contentTable);
+  await patientlyWaitFor(() => expect(getByText('Description')).toBeInTheDocument());
+  expect(queryByText('Content view')).toBeInTheDocument();
+  expect(queryByText('Type')).toBeInTheDocument();
+  expect(queryByText('Last published')).toBeInTheDocument();
+  expect(queryByText('Repositories')).toBeInTheDocument();
+  expect(queryByText('Synced to smart proxy')).toBeInTheDocument();
```

Comment:
these `queryByText` can be `getByText` if they are expected to be in the document, it will show a better error message if it's not present.